channels-rke2.yaml

```diff
@@ -1267,13 +1283,29 @@ releases:
         repo: rancher-rke2-charts
         version: 1.7.101
     featureVersions: *featureVersions-v1
+  - version: v1.25.11+rke2r1
+    minChannelServerVersion: v2.7.5-alpha1
+    maxChannelServerVersion: v2.7.99
+    serverArgs: *serverArgs-v1-24-2-rke2r1
+    agentArgs: *agentArgs-v1-25-7-rke2r1
+    charts: &charts-v1-25-11-rke2r1
+      <<: *charts-v1-25-10-rke2r1
+      rke2-canal:
+        repo: rancher-rke2-charts
+        version: v3.25.1-build2023051201
+      rke2-coredns:
+        repo: rancher-rke2-charts
+        version: 1.24.002
+      harvester-cloud-provider:
+        repo: rancher-rke2-charts
+        version: 0.2.200
   - version: v1.26.5+rke2r1
     minChannelServerVersion: v2.7.5-alpha1
     maxChannelServerVersion: v2.7.99
     serverArgs: *serverArgs-v1-24-2-rke2r1
     agentArgs: *agentArgs-v1-25-7-rke2r1
-    charts: &charts-v1-26-4-rke2r1
-      <<: *charts-v1-25-9-rke2r1
+    charts: &charts-v1-26-5-rke2r1
+      <<: *charts-v1-25-10-rke2r1
```

Comment:
Are we changing already published versions chart reference?

Reply:
It looks like yaml anchors for chart blocks were mis-named in last month's patch releases. I created another branch that removes those changes and ran go generate, and the data did not change. https://github.com/brooksn/kontainer-driver-metadata/compare/rke2-2023-june...brooksn:kontainer-driver-metadata:rke2-2023-june-undo-anchor-renaming?expand=1

Also, I just made a push that updated one more anchor name.